### PR TITLE
electron_sf_id_mediumplushltsafe_ZA added

### PR DESCRIPTION
**Pull request by alesaggio**

### CODE REVIEW

File: test/HtoZAConfiguration.py [hunk 1]
```diff
@@ -94,6 +97,8 @@
 # framework.getProducer('jets').parameters.cut = cms.untracked.string("pt > 20")
 #framework.getProducer('jets').parameters.computeRegression = cms.untracked.bool(True)
 
+framework.getProducer('electrons').parameters.scale_factors.id_mediumplushltsafe_ZA = cms.untracked.FileInPath('cp3_llbb/ZAAnalysis/data/ScaleFactors/Electron_MediumPlusHLTSafeID_moriond17.json')
```
Comment (OlivierBondu) [inline test/HtoZAConfiguration.py]: why the `_ZA` suffix for the variable ?

**Review by OlivierBondu — COMMENTED** [review]
thanks !

File: test/HtoZAConfiguration.py (hunk 1 above)
Comment (alesaggio) [reply test/HtoZAConfiguration.py]: Well, because it's for the ZA Analysis... should I remove it?

Comment (OlivierBondu) [reply test/HtoZAConfiguration.py]: the SF themselves are not specific to ZA ?

**Pull request merged**